pmoves/services/messaging-gateway/platforms/telegram.py

```
@@ -0,0 +1,208 @@
+
+        # Get bot info to verify token
+        try:
+            r = await self._client.get(
+                f"https://api.telegram.org/bot{self.bot_token}/getMe"
+            )
+            if r.status_code == 200:
+                bot_info = r.json()
+                logger.info(f"Telegram bot initialized: {bot_info.get('result', {}).get('username')}")
+            else:
+                logger.warning(f"Failed to verify Telegram bot: {r.status_code}")
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)